lib/v2/screens/receive_screens/receive_enter_data/interactor/mappers/create_invoice_result_mapper.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/page_commands.dart';
+import 'package:seeds/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/receive_enter_data_state.dart';
+
+class CreateInvoiceResultMapper extends StateMapper {
+  ReceiveEnterDataState mapResultToState(ReceiveEnterDataState currentState, Result result) {
+    if (result.isError) {
+      print('Error invoice hash not retrieved');
+      return currentState.copyWith(
+        pageState: PageState.success,
+        pageCommand: ShowTransactionFail(),
```

Comment:
@gguijarro-c-chwy  This page command is very common, should we make it global? or not? Because also they could vary in content. These classes are currently empty.
![image](https://user-images.githubusercontent.com/42857405/121564178-c763dd00-c9e0-11eb-9f2e-d95ff9aab974.png)

Reply:
i think we should have it global, perhaps the base PageCommand class has a few global commands 
ShowTransactionFail
ShowMessage(String meesage)
NavigateToRoute(Route route)

Reply:
You mean creating named constructors on the PageCommad class or just creating these classes in the same file?

Reply:
Ill have a pr up for this soon